front-end/cmb2-front-end-submit.php

```diff
@@ -154,7 +154,7 @@ function wds_do_frontend_form_submission_shortcode( $atts = array() ) {
 function wds_handle_frontend_new_post_form_submission() {
 
 	// If no form submission, bail
-	if ( empty( $_POST ) || ! isset( $_POST['submit-cmb'], $_POST['object_id'] ) ) {
+	if ( empty( $_POST ) || ! isset( $_POST['submit-cmb'], $_POST['object_id'] ) || $_POST['object_id'] != 'fake-oject-id' ) {
```

Comment:
Typo `'fake-oject-id'` should be `'fake-object-id'`. Otherwise lgtm

Reply:
The typo 'fake-oject-id' is even in [cmb2-front-end-submit.php](https://github.com/CMB2/CMB2-Snippet-Library/blob/master/front-end/cmb2-front-end-submit.php). That's why I used it here.

https://github.com/CMB2/CMB2-Snippet-Library/blob/4a0b2757894a3f9a6b1e91651a11594a6e6ad2d0/front-end/cmb2-front-end-submit.php#L98
https://github.com/CMB2/CMB2-Snippet-Library/blob/4a0b2757894a3f9a6b1e91651a11594a6e6ad2d0/front-end/cmb2-front-end-submit.php#L163